howto-website.md

```diff
@@ -10,7 +10,7 @@ This site is maintained by the HSF GitHub [contributors](https://github.com/orgs
 
 ## Implementation
 
-This website is implemented using [GitHub's Pages](https://pages.github.com/) service which makes it easy to create a website associated with a GitHub account or project. Pages uses [Jekyll](https://help.github.com/articles/using-jekyll-with-pages/), a tool to automatically build a website from source files (which are kept in GitHub). It supports structured sites like blogs in a simple but powerful way.
+This website is implemented using [GitHub's Pages](https://pages.github.com/) service, which makes it easy to create a website associated with a GitHub account or project. Pages uses [Jekyll](https://help.github.com/articles/using-jekyll-with-pages/), a tool to automatically build a website from source files (which are kept in GitHub). It supports structured sites like blogs in a simple but powerful way.
```

Comment:
Is "Pages" a tool - sorry for the ignorance? Otherwise it would be "Pages use".

Reply:
Yeah, it's a service that GitHub offer <https://pages.github.com/> so the grammar is correct here